### DISCUSSION TITLE
Should move_window respect taskbar placement on Windows?

### DISCUSSION THREAD

**Discussion by tanzim** [tauri-apps/tauri#7357] [Ideas]
Hi,

Firstly, thanks for a brilliant project.

I noticed that on Windows, using move_window(Position::BottomRight) will move the window to absolute bottom right, with the window appearing on top of the taskbar. I suspect that most users would probably want the extra refinement of being able to place the window above the taskbar and maybe even with some offset as that's how Windows 11 notification/connection center windows are displayed (see below). I wonder if this is something that should make it's way into Tauri core?
Thanks.

![image](https://github.com/tauri-apps/tauri/assets/1591642/b0d2cd77-9c49-462d-87ea-9972844ca379)

**Comment by FabianLars** [comment]
Are you talking about the positioner plugin? Because if so, then please open a feature request for it at the plugins-workspace repo. If you really meant tauri core then no, core should not be concerned about that imho.

**Reply by tanzim** [reply]
Yikes! You're right - I noticed that it's a feature of the positioner plug-in. Sorry about that.
